src/current/_includes/metrics-usage/txn.restarts.unknown.md

```diff
@@ -0,0 +1 @@
+The errors tracked by this metric are generally due to deadlocks. Deadlocks can often be prevented with a considered transaction design. Identify the conflicting transactions involved in the deadlocks, then, if possible, redesign the business logic implementation prone to deadlocks.
```

Comment:
suggest making 'conflicting transaction' a link to our transaction conflict docs you've linked from other metrics

last sentence is a bit run-on

suggest a small change to break it up like

"Identify the conflicting transactions involved in the deadlocks. Then, if possible, redesign the business logic implementation prone to deadlocks."

Reply:
added link to: https://www.cockroachlabs.com/docs/stable/performance-best-practices-overview#transaction-contention

fixed run-on sentence